cmd/cloudx/project/get_test.go

```diff
@@ -16,43 +15,32 @@ import (
 )
 
 func TestGetProject(t *testing.T) {
-	configDir := testhelpers.NewConfigDir(t)
-	cmd := testhelpers.ConfigAwareCmd(configDir)
-	email, password := testhelpers.RegisterAccount(t, configDir)
-
-	project := testhelpers.CreateProject(t, configDir)
-
 	t.Run(fmt.Sprintf("is able to get project"), func(t *testing.T) {
```

Comment:
Looks like you removed tests for `get identity-config` and did not add tests for `get permission-config`. Is that on purpose or accident?

Reply:
Some tests were moved to `cloudx/client`, but yeah I'll have to add  tests for permission config.